src/Command/ConsumeCommand.php

```diff
@@ -47,8 +47,21 @@ public function configure()
      */
     public function execute(InputInterface $input, OutputInterface $output)
     {
-        $queue = $this->queues->create($input->getArgument('queue'));
+        $queue = $this->getQueue($input->getArgument('queue'));
 
         $this->consumer->consume($queue, $input->getOptions());
     }
+
+    /**
+     * @param string $queue
+     * @return Queue
+     */
+    protected function getQueue($queue)
+    {
+        if (strpos($queue, ',') !== false) {
+            $queues = array_map([$this->queues, 'create'], explode(',', $queue));
+            return new \Bernard\Queue\RoundRobinQueue($queues);
```

Comment:
My purpose in referencing the FQCN here was to avoid loading the class if it wasn't used, but if it's your preference that I include a `use` statement and use its short name here anyway, I can do that.